substrate/frame/metadata-hash-extension/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "frame-metadata-hash-extension"
+version = "0.1.0"
+edition = "2021"
```

```suggestion
edition.workspace = true
```
CI is requiring License and Description as well.